app/helpers/think_feel_do_dashboard/application_helper.rb

```diff
@@ -2,5 +2,43 @@ module ThinkFeelDoDashboard
   # Ensure font awesome icon helpers are available.
   module ApplicationHelper
     include FontAwesome::Rails::IconHelper
+
+    TOP_LEVEL_CONTROLLERS = %w( arms groups participants users reports )
+
+    # Render navigational information in the form of breadcrumbs
+    def breadcrumbs
+      return unless show_breadcrumbs?
+
+      content_for(
+        :breadcrumbs,
+        content_tag(:ol, class: "breadcrumb") do
+          concat(content_tag(:li, link_to("Home", root_path)))
+
+          if can_view_resource_index?
+            concat(content_tag(:li,
+                               link_to(controller_name.capitalize,
+                                       url_for(controller: controller_name))
+                              )
+                  )
+          end
+        end
+      )
+    end
+
+    private
+
+    def show_breadcrumbs?
```

Comment:
Is this the preferred Rubocop indentation? I can never tell what it wants from me.